opm/simulators/wells/SingleWellState.cpp

```diff
@@ -245,31 +245,28 @@ void SingleWellState::update_injector_targets(const Well& ecl_well, const Summar
         return;
     }
 
-
-    if (inj_controls.cmode == Well::InjectorCMode::GRUP) {
-        this->bhp = this->perf_data.pressure_first_connection * bhp_safety_factor;
-        return;
+    // we initialize all open wells with some rate to avoid singularities
+    double inj_surf_rate = 0.0001;
```

Comment:
In this part of the code everything is SI so 1.0e-4 corresponds to 8.64 SM3/day (or 54.344 STB/day for that matter).  Is that a reasonable guess?

Reply:
As long as it is >0 it does not seem to be very sensitive to this number. For producers 1000 SM3/day is used. We could use the same for injectors.

Reply:
@bska Can this be merged? Do you want me to change the initial guess to 1000 SM3/day?

Reply:
> without hitting the procedure with a large hammer

I can hit it with a small hammer if you like. Any value  > epsilon should work. But 1000 SM3DAY is used for the producers. So what is done here make the code for the injectors consistent with producers. Since the well fraction are computed from the initial rates a fix like this is needed if you want to run a case where your injector is bhp or group controlled from the start. If I remember correctly we used to initialize with an epsilon many years ago in the old wellState code in opm-core.